lib/approvals/writer.rb

```diff
@@ -16,16 +16,13 @@ module Writer
       html: Writers::HtmlWriter.new,
       hash: Writers::HashWriter.new,
       array: Writers::ArrayWriter.new,
+      txt: Writers::TextWriter.new,
```

Comment:
`:txt` is the default if no `:format` is set in https://github.com/kytrinyx/approvals/blob/master/lib/approvals/approval.rb#L30